lib/_stream_readable.js

```diff
@@ -558,7 +560,7 @@ function emitReadable_(stream) {
 function maybeReadMore(stream, state) {
   if (!state.readingMore) {
     state.readingMore = true;
-    process.nextTick(maybeReadMore_, stream, state);
+    nextTick(maybeReadMore_, stream, state);
```

Comment:
This should not work with the polyfill. I think our browser testing is adding `process.nextTick`.

Reply:
its just using the first if here https://github.com/medikoo/next-tick/blob/master/index.js#L47-L49 because airtap is injecting everything, its hard to validate this with the current repo setup i will try to fix this

Reply:
Yes exactly. However if those transform are not put in there, it won't work.